docs/source/rest-api/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _sdc-api:
+
+REST API
+========
+
+The SDC supports a REST API that allows users to upload and download files, as
+well as query for file metadata. The following documentation describes the
+various endpoints that are supported and how to use them.
+
```

Comment:
Since this just confused me, could you add the dev API url (`https://api.dev.imap-mission.com`) so in the near future instrument teams can use this as a reference?